.squash.yml

```diff
@@ -1,14 +1,14 @@
 deployments:
   default:
-    dockerimage: python:3.7.4-stretch
+    dockerimage: python:3.10.5-slim-buster
     build_steps:
       - apt-get update && apt-get install -y libssl-dev libpq-dev git build-essential libfontconfig1 libfontconfig1-dev curl
       - RUN bash -c "curl -sL https://deb.nodesource.com/setup_16.x | bash -"
       - apt install -y nodejs
       - pip install setuptools pip --upgrade --force-reinstall
       - cd /code
     post_build_steps:
-      - npm install --no-save --no-audit --progress=false
+      - npm install --no-save --no-audit
```

Comment:
Not sure if it's helpful but `npm ci` is probably more suitable here. 

https://docs.npmjs.com/cli/v8/commands/npm-ci

Reply:
Also. It's odd you needed to update the package lock version. Pretty sure we are using node 16 which should be using V2 already.

Reply:
> Not sure if it's helpful but `npm ci` is probably more suitable here.

Yes, I think you're right, we should be using `npm ci`. I was under the mistaken impression that `npm ci` didn't install `devDependencies`, which I thought would break the build, but it does, actually. I'll try switching to that and see if it helps.


> Also. It's odd you needed to update the package lock version. Pretty sure we are using node 16 which should be using V2 already.

Yeah, we are on 16 now, but apparently we never updated the package-lock.json format.